model/nc_news.model.js

```diff
@@ -75,31 +76,64 @@ exports.insertCommentbyArticle_id = (article_id, body) => {
       msg: "bad request!",
     });
   }
-  return db
-    .query(`SELECT * FROM articleS WHERE article_id = $1`, [article_id])
-    .then((result) => {
-      if (result.rows.length === 0) {
-        return Promise.reject({
-          status: 404,
-          msg: "article not found",
-        });
-      }
-      if (!body.body || !body.author) {
-        return Promise.reject({
-          status: 400,
-          msg: "all inputs are required",
-        });
-      }
-      return db
-        .query(
-          `
-          INSERT INTO comments
-          (body,author,article_id)
-          VALUES ($1,$2,$3) RETURNING *;`,
-          [body.body, body.author, article_id]
-        )
-        .then((result) => {
-          return result.rows[0];
-        });
+  if (!body.body || !body.author) {
+    return Promise.reject({
+      status: 400,
+      msg: "all inputs are required",
+    });
+  }
+  return Promise.all([
+    checkIfArticleExists(article_id),
+    checkIfUserExists(body.author),
+  ]).then(() => {
+    return db
+      .query(
+        `
+        INSERT INTO comments
+        (body,author,article_id)
+        VALUES ($1,$2,$3) RETURNING *;`,
+        [body.body, body.author, article_id]
+      )
+      .then((result) => {
+        return result.rows[0];
+      });
+  });
+};
+
+exports.updateArticulebyArticule_id = (inc_votes, article_id) => {
+  const incVotes = inc_votes;
+  if (isNaN(article_id)) {
+    return Promise.reject({
+      status: 400,
+      msg: "bad request!",
     });
+  }
```

Comment:
You can leave this for now, but like I mentioned in a previous PR, you don't need this check, because postgres will throw an error automatically when you try to insert something that isn't a number into a field that requires a number.